extensions/applicationinsights-osplugin-js/tslint.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
I think we can delete this file now as we (should) be using eslint by default. This is a left over of the previous linting code we used.